assets/src/scripts/components/map/nldiMapping.js

```diff
@@ -0,0 +1,152 @@
+import { geoJson, circleMarker } from 'leaflet';
+import { get } from '../../ajax';
+import config from '../../config';
+import { select } from 'd3-selection';
+
+/**
+ * Add NLDI layer overlays to a leaflet map. An overlay is added for the flowlines
+ * upstream and downstream of a site; another overlay is added to upstream and
+ * downstream NWIS sites. Pop-ups are created for each feature in the overlay
+ * layers.
+ *
+ * @param {L.map} map The leaflet map to which the overlay should be added
+ * @param {L.Control} legendControl The map's legend control
+ * @param {String} sitno The starting site for navigation
+ */
+export const addNldi = function(map, legendControl, siteno) {
+    const nldiUrl = config.NLDI_SERVICES_ENDPOINT;
+    const featureSource = 'nwissite';
+    const dataSource = 'nwissite';
+    const upstreamNavigation = 'UM';
+    const downstreamNavigation = 'DM';
+    const featureId = 'USGS-' + siteno;
+    const distanceParam = '?distance=322';
```

Comment:
I'd like to see the distance to be a configuration parameter.  So that would go in server/config.py. Can probably just set it there, maybe Initial_Upstream_Downstream_Distance.

Then you could set this distanceParam=`$distance=${config.NLDI_SERVICES_ENDPOINT}`. Uses template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals